ibm/service/power/data_source_ibm_pi_datacenter.go

```diff
@@ -28,6 +29,124 @@ func DataSourceIBMPIDatacenter() *schema.Resource {
 			},
 
 			// Attributes
+			Attr_CapabilityDetails: {
```

Comment:
That's really rough. Did you ever get a chance to ask esdras to see if they could cut out one of the middle layers?

Reply:
That was updated.